d2bs/kolbot/libs/common/Town.js

```diff
@@ -67,13 +67,18 @@ var Town = {
 
 	// Do town chores
 	doChores: function (repair = false) {
+		var i,
+			cancelFlags = [0x01, 0x02, 0x04, 0x08, 0x14, 0x16, 0x0c, 0x0f, 0x19, 0x1a],
+			npc = getUnit(1, "warriv");
```

Comment:
Please use NPC.Warriv variable for npc name to ensure compatibility with different locales.